backend/routes/PostReaction/Like.py

```
@@ -0,0 +1,79 @@
+from fastapi import APIRouter, Depends, HTTPException, Query
```

Ruff (0.8.2)

`fastapi.Query` imported but unused

Remove unused import: `fastapi.Query`

(F401)